backend/app/controllers/users_controller.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+class UsersController < ApplicationController
```

Comment:
Ohhhh. Right. I was thinking that after creating, user can automatically login 😓

Reply:
Ah I see. in that case put that `create user + create jwt token` in a registrations controller. sa "signup" endpoint mo. Wag dito. 

Because this looks like a generic UsersController. 

If you want, retrofit this controller into a Registrations controller. That way its clear that it is specifically made for registrations.